### ISSUE TITLE
Dropbox oAuth v1 has been shut down on 2017 September 28th

### ISSUE THREAD

**Issue by mmattel** [owncloud/files_external_dropbox#12]
https://blogs.dropbox.com/developers/2017/09/api-v1-shutdown-details/

```
As previously announced, API v1 is being retired on Thursday, September 28th, 2017.

On this date, API v1 endpoints will return a 400 error with this message: 
{“error”: “v1_retired”}. This means any integrations or applications still relying on 
API v1 endpoints may stop working.
```

Can prove, my Dropbox connection with oAuth1 is not longer working...

**Comment by patrickjahns** [comment]
@mmattel 
This repository already contains the updated version for dropbox access. 
Released here: 
https://marketplace.owncloud.com/apps/files_external_dropbox

Release Announcement: 
https://owncloud.org/blog/external-storage-dropbox/

**Comment by mmattel** [comment]
Thanks,

just to let you know:
![image](https://user-images.githubusercontent.com/3321281/31229152-128bbf70-a9e0-11e7-9410-ce55556e4d75.png)
The new version (disabled) has the same name as the old one (enabled) which is somehow confusing...

**Comment by DeepDiver1975** [comment]
> You also have in the available storages two entries you can´t distinguish...

this will be fixed with the next release - fix is here #11

**Comment by mmattel** [comment]
Sorry for the stupid question, but is from the ownCloud release announcement 
``#10 Fill the details Client Id, Client Secrets from you Dropbox App page``
matching the ``App Key / App Secret`` from the dropbox page?`
I guess so as there is nothing else...

I entered all but still get a red square...
No negative feedack from Dropbox
If I try to enter a false url than it complains so communication works

**Comment by PVince81** [comment]
@mmattel is this related to https://github.com/owncloud/files_external_dropbox/issues/13. Close in favor of https://github.com/owncloud/files_external_dropbox/issues/13 ?

**Comment by mmattel** [comment]
It is indirect related with #13 
I opend this because oAuth1 was shut down by DB and there are troubles with signing in with oAuth2 (#13)
The intention of this issue was to document this for all those who did not recognize the shutdown and were wondering about the non functional implemetation of the old DB connection...
Imho, this issue can be closed when #13 has been resolved and published.
Please also consider to have the text at the market place updated so readers clearly identify the need for this app to work with DB.

**Comment by patrickjahns** [comment]
Closing - 1.0.1 release is on marketplace
